aqt/helper.py

```diff
@@ -210,6 +210,8 @@ def get_hash(archive_path: str, algorithm: str, timeout: Tuple[float, float]) ->
             if len(_hash) == hash_lengths[algorithm]:
                 return binascii.unhexlify(_hash)
         except (ArchiveConnectionError, ArchiveDownloadError, binascii.Incomplete, binascii.Error):
+            if one_rep:
+                break
```

Comment:
This is so that, if the extensions are checked silently, they cannot try all the mirrors. This is not the idea solution, but invert the caller would be longer.

Why is this issue? The caller is basically spamming the servers: at list-qt, it pulls the webpage for each extension. But downstream, for each of those, get_hash can pull it as many times as there is mirrors. If the download error happens because the client entered a bad arch, then this causes aqt to freeze. It tries to pull nb_extensions * nb_mirrors, with each time a timeout. Do this a few time and you get timeed out by major mirror for too many calls in a short time